.vscode/settings.json

```diff
@@ -8,40 +8,49 @@
   "files.insertFinalNewline": true,
   "[python]": {
     "editor.tabSize": 4,
-    "editor.rulers": [120]
```

Comment:
It looks like I added this in a previous PR. I didn't want to change around anything for python so just removing it again.